.github/dependabot.yml

```diff
@@ -18,6 +18,11 @@ updates:
       separator: "-"
     labels:
       - "sdou"
+    allow:
+      - dependency-name: "org.liquibase:liquibase-secure"
```

Comment:
@abrackx I can't keep it straight. Are we publishing to maven central? If so, isn't the target going to be com.liquibase?